### PR TITLE
Fix message test flow

### PR DESCRIPTION
**Pull request by dannytranlx**
Latest build on master wasn't compatible with the Heroku deploy due to:
```
2024-03-04T21:28:56.837216+00:00 app[web.1]: Error: Cannot find module './TokenUtils'
```

This PR fixes the typo and the message test flow when we migrated from smooch-js in #15 

And finally added linting.

#18 will also be resolved when we will upgrade the stack to Heroku-22 and NodeJS > 8.

Reference: https://help.heroku.com/ZBZB0LNB/let-s-encrypt-s-root-certificate-change-and-your-heroku-app

### CODE REVIEW

File: smoochApi.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const SunshineConversationsClient = require('sunshine-conversations-client');
 
 const config = require('./config');
-const {extractAppId} = require("./TokenUtils");
+const { extractAppId } = require('./tokenUtils');
```
Comment (dannytranlx) [inline smoochApi.js:4]: `TokenUtils` -> `tokenUtils`

File: smoochApi.js [hunk 2]
```diff
@@ -10,44 +10,91 @@ function getOrCreateUser(appId) {
...
+          },
+        });
+
+      return apiInstance.postMessage(appId, conversation.id, messagePost);
     })
     .then(() => {
       return appUser;
     });
-}
+};
```
Comment (dannytranlx) [inline smoochApi.js:100]: Updated test message flow

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 2,
```
Comment (a-brun) [inline .prettierrc:2]: Why not reuse the sunco one?

Comment (dannytranlx) [reply .prettierrc:2]: I didn't wanted to disrupt the code base too much (especially tabWidth: 4 on sunco), just needed a bit of consistency on single quotes and semi-colons

File: smoochApi.js [hunk 4]
```diff
@@ -10,44 +10,91 @@ function getOrCreateUser(appId) {
   const userId = 'test@shoplifter.com';
   const apiInstance = new SunshineConversationsClient.UsersApi();
 
-  return apiInstance.getUser(appId, userId)
+  return apiInstance
+    .getUser(appId, userId)
     .catch((err) => {
       if (!err.response || err.response.status !== 404) {
```
Comment (a-brun) [inline smoochApi.js:16]: Nothing to do with your PR, but I'm seeing `response.status` as a function. 🤔  I'm surprised this ever worked.

**Review by a-brun — APPROVED** [review]

**Pull request merged**